src/query/execute_schema.c

```diff
@@ -2924,7 +2931,7 @@ create_or_drop_index_helper (PARSER_CONTEXT * parser, const char *const constrai
 
       // Class or shared attributes are not considered. These are not indexed columns.
       // Also, The prefix index is also not supported.(The prefix index  will be deprecated.)
-      if (ctype == DB_CONSTRAINT_INDEX || ctype == DB_CONSTRAINT_REVERSE_INDEX)
+      if (ctype == DB_CONSTRAINT_INDEX || ctype == DB_CONSTRAINT_REVERSE_INDEX || ctype == DB_CONSTRAINT_VECTOR_INDEX)
```

Comment:
vector index는 deduplicated key 기능과 무관
```suggestion
      if (ctype == DB_CONSTRAINT_INDEX || ctype == DB_CONSTRAINT_REVERSE_INDEX)
```

Reply:
수정하였습니다.